__tests__/Unit/Components/Tasks/Card.test.tsx

```diff
@@ -124,7 +122,8 @@ describe('Task card', () => {
         expect(queryByTestId('task-card')).toBeInTheDocument();
     });
 
-    test('should show edit button when ALT key is long pressed', () => {
+    test('should show edit button when ALT key is long pressed', async () => {
+        server.use(...adminUserHandler);
```

Comment:
Why are we using aminUserHandler here when we are not doing any testing related to it?

Reply:
Beacuse the code is written as `{isUserAuthorized && showEditButton && <EditButton />}` . So to get the isUserAuthorized values as true we need to use adminUserHandler